static/js/tools/choropleth/choropleth.tsx

```diff
@@ -576,6 +608,74 @@ class ChoroplethMap extends Component<PropsType, StateType> {
     this.mouseOutAction(index);
   };
 
+  /**
+   * Redirects the webclient to a particular geo. Handles breadcrumbs in redirect.
+   * @param {string} geoDcid to redirect to.
+   * @param {string} curGeo human-readable current geo, e.g. United States when geoDcid
+   *                 is country/USA.
+   */
+  private redirectToGeo = (geoDcid: string, curGeo: string): void => {
+    const url = new URL(window.location.href);
+    const currGeoId = url.searchParams.get("geoDcid");
+    let baseUrl = "/tools/choropleth";
```

Comment:
js string template would be better:

`/tools/choropleth${buildChoroplethParams()}&geoDcid=${geoDcid}&bc=`

Reply:
you could also use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)